src/site/content/en/blog/nfc/index.md

```diff
@@ -263,11 +268,64 @@ const ndef = new NDEFReader();
 ndef.write("Writing data on an empty NFC tag is fun!", { overwrite: false })
 .then(() => {
   console.log("Message written.");
-}).catch(_ => {
+}).catch(error => {
   console.log(`Write failed :-( try again: ${error}.`);
 });
 ```
 
+### Make NFC tags read-only {: #make-read-only }
+
+To prevent malicious users from overwriting NFC tags content, it is possible to
+make NFC tags permanently read-only. This operation is a one-way process and
+cannot be reversed. Once an NFC tag has been made read-only, it can't be written
+anymore.
+
+To make NFC tags read-only, first instantiate a new `NDEFReader` object. Calling
+`makeReadOnly()` returns a promise. The [user may be prompted] if access was not
+previously granted. The promise will resolve if the following conditions are all
+met:
+
+- It must be called in response to a user gesture such as a touch gesture or
+  mouse click.
+- User has allowed the website to interact with NFC devices when they tap their
+  phone.
```

Comment:
Tap there phone where? Maybe just drop that part

Reply:
You're right. Let's make it simpler. Fixed in https://github.com/GoogleChrome/web.dev/pull/7252/commits/fe3c0e82929b7745d9e49a0b6ce3aa39df59c5cd